lib/foreman_monitoring/engine.rb

```diff
@@ -86,14 +86,14 @@ class Engine < ::Rails::Engine
     end
 
     config.to_prepare do
-      begin
+      
```

Comment:
No need to add an empty line. And if you're dropping this level of indenting you should also dedent the actual lines.